### PR TITLE
Add minimal_degree_of_autonomy parameter to all constraints.csv files

### PR DESCRIPTION
**Pull request by MaGering**
This PR fixes the mentioned todo in the first checkbox of Issue https://github.com/greco-project/pvcompare/issues/211.

The parameter `minimal_degree_of_autonomy` is added to all `constraints.csv` files in pvcompare and set to `0`.
It was supposed to be an optional parameter, which is set to its default value, if not provided. Due to a bug however, described in [this mvs-issue](https://github.com/rl-institut/multi-vector-simulator/issues/780), `minimal_degree_of_autonomy` is a required parameter. According to @smartie2076, it is not going to be fixed soon, which is why we include it on our `constraints.csv` files with the upcoming release of MVS.

This PR can be merged either before or after the update of pvcompare to MVS v0.5.5. The simulations run either way.
If it is merged before the update, we'll get this `WrongParameterWarning` running the simulations:
```
WrongParameterWarning: The parameter minimal_degree_of_autonomy in the file/home/local/RL-INSTITUT/marie-claire.gering/Repositories/pvcompare/examples/example_user_inputs/mvs_inputs_sector_coupling/csv_elements/constraints.csv is not expected. 
Expected parameters are ['minimal_renewable_factor', 'maximum_emissions']
  f"The parameter {i} in the file"
```

The following steps were realized, as well (if applies):
- :x: Use in-line comments to explain your code
- :x: Write docstrings to your code
- :x: For new functionalities: Explain in readthedocs
- :x: Write test(s) for your new patch of code
- :x: Update the CHANGELOG.md (let's start this after the first release)
- :x: Apply black (`black . --exclude docs/`)
- [x] Check if full simulation tests pass locally (`EXECUTE_TESTS_ON=master pytest`)

*Please mark above checkboxes as following:*
- [ ] Open
- [x] Done

:x: Check not applicable to this PR

<sub>*For more information on how to contribute check the [CONTRIBUTING.md](https://github.com/rl-institut/mvs_eland/blob/dev/CONTRIBUTING.md).*<sub>

### CODE REVIEW

**Review by SabineHaas — APPROVED** [review]
Thank you @MaGering!

- there's a typo in factor in the examples

To my mind you can already merge it as we are aware of the warning. If @Piranias agrees, as well, of course.

File: examples/example_user_inputs/mvs_inputs_electricity_sector/csv_elements/constraints.csv [hunk 1]
```diff
@@ -1,3 +1,4 @@
 ,unit,constraints
 minimal_renewable_factor,factor,0
 maximum_emissions,kgCO2eq/a,None
+minimal_degree_of_autonomy,facotr,0
```
Comment (MaGering) [inline examples/example_user_inputs/mvs_inputs_electricity_sector/csv_elements/constraints.csv:4]:
```suggestion
minimal_degree_of_autonomy,factor,0
```

File: examples/example_user_inputs/mvs_inputs_sector_coupling/csv_elements/constraints.csv [hunk 2]
```diff
@@ -1,3 +1,4 @@
 ,unit,constraints
 minimal_renewable_factor,factor,0
 maximum_emissions,kgCO2eq/a,None
+minimal_degree_of_autonomy,facotr,0
```
Comment (MaGering) [inline examples/example_user_inputs/mvs_inputs_sector_coupling/csv_elements/constraints.csv:4]:
```suggestion
minimal_degree_of_autonomy,factor,0
```

File: examples/example_user_inputs/mvs_inputs_sector_coupling_gas/csv_elements/constraints.csv [hunk 3]
```diff
@@ -1,3 +1,4 @@
 ,unit,constraints
 minimal_renewable_factor,factor,0
 maximum_emissions,kgCO2eq/a,None
+minimal_degree_of_autonomy,facotr,0
```
Comment (MaGering) [inline examples/example_user_inputs/mvs_inputs_sector_coupling_gas/csv_elements/constraints.csv:4]:
```suggestion
minimal_degree_of_autonomy,factor,0
```

**Review by Piranias — APPROVED** [review]

**Pull request merged**